src/mlpack/core/tree/ballbound.hpp

```diff
@@ -157,7 +157,8 @@ class BallBound
   /**
    * Calculates minimum and maximum bound-to-point distance.
    *
-   * @param point Point to which the minimum and maximum distances are
+   * @param other Bound to check the condition.
+   * @param * Point to which the minimum and maximum distances are
```

Comment:
Wasn't sure about this one

Reply:
```suggestion
   * @param other Point to which the minimum and maximum distances are
```
I think just the name of the parameter was wrong.